tool/templates/c/runtime.h

```diff
@@ -16,13 +16,11 @@ typedef uint_least16_t char16_t;
 typedef uint_least32_t char32_t;
 #endif
 
+// START_SHARED_CAPI
```

Comment:
issue: I don't see value in doing this when we have jinja. I'm fine with this being in the same header, but let's use jinja to do this kind of thing

Reply:
The value is that this is a valid header file that builds and lints.

Reply:
We build and lint the examples anyway, I don't really think this is a problem?

Plus the hpp file won't build and lint since the h file isn't available.